packages/salesforce-adapter/src/filters/custom_objects_instances.ts

```diff
@@ -103,18 +105,33 @@ const buildQueryStrings = async (
   return buildSelectQueries(typeName, fieldNames, queryConditions)
 }
 
+type GetRecordsParams = {
+  client: SalesforceClient
+  typeAndFetchSettings: CustomObjectFetchSetting
```

Comment:
```suggestion
  settings: CustomObjectFetchSetting
```
I'd simplify the name as it's quite confusing

Reply:
The reason I chose this name is because it seemed weird that the function doesn't take the type to fetch as a parameter. So I wanted to make that more explicit. Do you have an idea on how I can make it clear from the function's signature that it receives the type as a parameter?